### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by 50North4West** [TheFrett/msfs_g36_project#14]
A while ago I made a Word/PDF version of the fuel / speeds as per the forum post (https://forums.flightsimulator.com/t/how-to-best-practices-to-fly-bonanza-g36/167175). Thought I would add it here. I'm happy to try to pull together a very limited sim POH. WHat do you think?

[Bonanza G36 Speed.docx](https://github.com/TheFrett/msfs_g36_project/files/5207329/Bonanza.G36.Speed.docx)
[Bonanza G36 Speed.pdf](https://github.com/TheFrett/msfs_g36_project/files/5207330/Bonanza.G36.Speed.pdf)

**Comment by TheFrett** [comment]
That would be awesome if you want to do that!

**Comment by 50North4West** [comment]
Any other thoughts for what to inlcude? We can cherry pick out of the main G36 manual depending on how far we want to go I suppose?

**Comment by 50North4West** [comment]
@TheFrett how about something like this for the documentation? Made a start...
[documentation.zip](https://github.com/TheFrett/msfs_g36_project/files/5217007/documentation.zip)

**Comment by 50North4West** [comment]
Note to Self:

File location to find Instrument ID's \Official\OneStore\fs-base-aircraft-common\ModelBehaviorDefs\Asobo\Common\Subtemplates

**Comment by 50North4West** [comment]
Hi All

I think it would be good to add information on how to correctly set-up the aircraft in the cruise etc. with regard the ROP/LOP settings. I've been following your amazing tuning of the flight model and really impressed.

I think us armchair flight dabblers might need a little help understanding it so thought it would be a good idea to add a section; do you know if there are any resources / youtube's that I can watch to get something written?

**Comment by TheFrett** [comment]
Are you following the discussion in the main thread? Gotta admit some of the things are beyond me. Have to dig the theory some time...

**Comment by 50North4West** [comment]
Yes been following; have learnt loads already, although all very new to me. Gotta read the POH some more and watch some YouTube videos! Think novice pilots (like me) will need some help in properly leaning the aircraft :-)

**Comment by TheFrett** [comment]
Merged your PR and had a look; 

Very nice, I like the visual style. Will read it in more detail later.

I do think however that we have to offer it separately and not as part of the community folder package...

**Comment by 50North4West** [comment]
Cool, got as far as the emergency parts. Takes a while as essentially just copying and re-writing from the POH and making it look nice and readable. 

Shall I host the documentation in a seperate Git and we can link it on the thread?? You mentioned a FAQ in the forums, do you think it would be useful in the documentation?? Happy to compile a list and add them in.

**Comment by TheFrett** [comment]
Ahh... I moved it one level up, see dev version. Would be better if we keep it in one place and there it does no harm (we can offer the documentation separately for a release).

Already had two questions here under discussion:

https://flightsim.to/file/1056/g36-improvement-project

We can add more from the forum and include it in the documentation maybe?

**Comment by 50North4West** [comment]
Got you! :-) much better.

Totally with you regard FAQ, will add it to near the install instructions - I'll edit the original post at the top with the most recent questions, perhaps use this as a place to pull them all together??
